bgpd/bgp_ecommunity.h

```diff
@@ -347,7 +348,7 @@ static inline void encode_color(uint32_t color_id, struct ecommunity_val *eval)
 	memset(eval, 0, sizeof(*eval));
 	eval->val[0] = ECOMMUNITY_ENCODE_OPAQUE;
 	eval->val[1] = ECOMMUNITY_COLOR;
-	eval->val[2] = 0x00;
+	eval->val[2] = (as << 6) & 0xff;
```

Comment:
Where is this _NEW_ encoding defined? Could you point to the right RFC (section)?

Reply:
In this sectino：8.8.1.  Color-Only BGP Destination Steering

Reply:
I don't see any paragraph saying about this encoding, or am I blind?

Reply:
Mybe the description is not very clear,  similar to nokia,huawei,arista,cisco

Arista: https://arista.my.site.com/AristaCommunity/s/article/SR-TE-Policy-With-Null-Endpoint-Color-Only-IP-Steering-Using-CO-Bits-Modification
nokia:https://documentation.nokia.com/acg/23-7-2/books/classic-cli-part-i/c227-bgp-sr-policies.html#ai9j1gjko5__ai9j1gjko7
huawei:https://support.huawei.com/enterprise/en/doc/EDOC1100143014/c38497ba/sr-mpls-te-policy
cisco:https://www.cisco.com/c/en/us/td/docs/iosxr/cisco8000/segment-routing/78x/b-segment-routing-cg-cisco8000-78x/configuring-sr-te-policy.html#concept_lzh_xyn_rkb

Reply:
Ok, now I see. This is called CO flags. 

While this is fine, I find the naming confusing. Why these flags are called `as`?